includes/common.php

```diff
@@ -69,8 +78,11 @@ function stripslashes_gpc(&$value)
 ini_set('error_log', 'includes/error.log');
 
 require 'includes/GeneralFunctions.php';
-set_exception_handler('exceptionHandler');
-set_error_handler('errorHandler');
+
+if (!isCommandLineInterface()) {
+	set_exception_handler('exceptionHandler');
+	set_error_handler('errorHandler');
+}
```

Comment:
We should add set_exception_handler for cli-mode otherwise you will be get unreadable error message :-)